src/Features/CSharp/Portable/InlineHints/CSharpInlineParameterNameHintsService.cs

```diff
@@ -100,5 +100,10 @@ protected override bool IsIndexer(SyntaxNode node, IParameterSymbol parameter)
         {
             return node is BracketedArgumentListSyntax;
         }
+
+        protected override string HintText(string parameterName)
+        {
+            return parameterName + ": ";
```

Comment:
parameterName seems unnecessary in this comptuation.  we can just have:

`private readonly string _hintTextSuffix;`

and then just have the abstract base just concat this to the parameter name there.